.github/workflows/common_pull_request.yaml

```diff
@@ -152,6 +158,12 @@ jobs:
         password: ${{ secrets.CONTAINER_REPO_TOKEN }}
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set ownership
+        run: |
+          # Fix for git not liking owner of the checkout dir
+          chown -R $(id -u):$(id -g) $PWD
```

Comment:
Is it worth defining a `nwx_checkout.yaml` workflow since these changes need to stay coupled?

Reply:
At this point, I really don't think so. If there gets to be more than a couple of things that are coupled together, maybe.